ansible_collections/arista/avd/docs/porting-guides/4.x.x.md

```diff
@@ -37,6 +37,28 @@ bgp_peer_groups:
 
 TODO: Add link to data model once schema is done.
 
+### Connected endpoints
+
+Removed keys:
+
+```yaml
+<connected_endpoints_keys.key>:
+  - adapters:
+      - server_ports:
+        port_channel:
+          esi: <str>
```

Comment:
so probably I would split this in two parts

1. server_ports
2. port_channel.esi

otherwise it can be easy to overlook one or the other or more difficult to understand all the implication (like why is port_channel here when it looks lthe same in replacement blah blah)

Reply:
Yeah I cannot find a trace of it for connected_endpoints port_channels indeed.

Reply:
I have removed it.